src/dlgProfilePreferences.cpp

```diff
@@ -88,129 +86,127 @@ dlgProfilePreferences::dlgProfilePreferences( QWidget * pF, Host * pH )
-                            pushButton_green->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mGreen.name() ) );
-                           pushButton_Lgreen->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightGreen.name() ) );
-                              pushButton_red->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mRed.name() ) );
-                             pushButton_Lred->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightRed.name() ) );
-                             pushButton_blue->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mBlue.name() ) );
-                            pushButton_Lblue->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightBlue.name() ) );
-                           pushButton_yellow->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mYellow.name() ) );
-                          pushButton_Lyellow->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightYellow.name() ) );
-                             pushButton_cyan->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mCyan.name() ) );
-                            pushButton_Lcyan->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightCyan.name() ) );
-                          pushButton_magenta->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mMagenta.name() ) );
-                         pushButton_Lmagenta->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightMagenta.name() ) );
-                            pushButton_white->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mWhite.name() ) );
-                           pushButton_Lwhite->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mLightWhite.name() ) );
-
-                 pushButton_foreground_color->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mFgColor.name() ) );
-                 pushButton_background_color->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mBgColor.name() ) );
-         pushButton_command_foreground_color->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mCommandFgColor.name() ) );
-         pushButton_command_background_color->setStyleSheet( QStringLiteral( "QPushButton{background-color: %1;}" ).arg( mpHost->mCommandBgColor.name() ) );
+    pushButton_command_line_foreground_color->setStyleSheet(QStringLiteral("QPushButton{background-color: %1;}").arg(mpHost->mCommandLineFgColor.name()));
```

Comment:
Pity about the loss of the indentation to line up on the assignment but _dem's da breaks_ I guess...